cmd/registry/patch/deployment.go

```diff
@@ -39,11 +39,27 @@ func relativeSpecRevisionName(apiName names.Api, spec string) string {
 }
 
 // optionalSpecRevisionName returns a spec revision name if the subpath is not empty
-func optionalSpecRevisionName(deploymentName names.Deployment, subpath string) string {
+func optionalSpecRevisionName(ctx context.Context, client *gapic.RegistryClient, deploymentName names.Deployment, subpath string) (string, error) {
```

Comment:
As this is quite a vague function name, we should make sure the comment fully documents what to expect.

Reply:
yes, it should probably be renamed. I'll suggest something in the next commit.

Reply:
`resolveSpecRevisionName` with a better comment.